thatkitebot/cogs/electronics.py

```diff
@@ -48,6 +48,40 @@ def slash_preprocessor(a: str):
     """
     return a.replace("v", "").replace("V", "").replace("u", "µ").replace("F", "").strip() if a else None
 
+def FAQ():
```

Comment:
not really a good naming scheme, this would make it seem like it is a class

Reply:
yea I kinda don't know how to name it

Reply:
don't  make it its own function the
just include it in the command function